crates/matrix-sdk-crypto/src/backups/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The Matrix.org Foundation C.I.C.
+// Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
We probably want to list both years instead replacing.

Reply:
Done in https://github.com/matrix-org/matrix-rust-sdk/pull/725/commits/422092e2807e86726a0d5b05563866eab91cb184.